auditlog/context.py

```diff
@@ -62,6 +66,7 @@ def _set_actor(user, sender, instance, signal_duid, **kwargs):
             and instance.actor is None
         ):
             instance.actor = user
+            instance.actor_email = user.email
```

Comment:
what happens if we have a custom user model without an `email` field? it has to be set to `None`